modules/osdk-pruning-library-about.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-pruning-library.adoc
+
+:_content-type: CONCEPT
+[id="osdk-about-pruning-library_{context}"]
+= About the operator-lib pruning library
+
+Objects, such as jobs or pods, are created as a normal part of the Operator life cycle. If these objects are not removed, they can stay in the cluster and consume resources.
+
+In the past, the following options were available for pruning unnecessary objects:
+
+* Operator authors had to create a unique pruning solution for their Operators
+* Cluster administrators had to clean up objects on their own
+
+The `operator-lib` pruning library allows Go-based Operators to clean up, or _prune_, objects when they are no longer needed. Operator authors can use the library to create custom hooks and strategies.
```

Comment:
"`operator-lib` pruning library" sounds a bit awkward to me but I'm struggling with offering a better phrasing. 

So `operator-lib` itself is a library. It contains utility functions that help you with pruning. But that definition does NOT help in phrasing it any better IMO haha so if it helps you rephrase, that's great. If not, then I would leave it as is since it is consistently used in this document.